### PR TITLE
iron-doc-viewer polish of the day

### PR DESCRIPTION
**Pull request by aomarks**
Live at https://aomarks-test-docs.appspot.com/app-layout.html#/elements/app-header-layout

### CODE REVIEW

File: iron-doc-nav.html [hunk 1]
```diff
@@ -124,8 +154,21 @@
...
+          return (path === item.path || path.indexOf(item.path + '/') === 0)
+              && item.demos.length > 0;
+        },
+
+        _trayHeight(item, path) {
+          if (!this._isExpanded(item, path)) {
+            return 0;
+          }
+          // line-height of each item + padding + borders;
+          return (item.demos.length * 30) + 40 + 2;
```
Comment (samuelli) [inline iron-doc-nav.html]: Hmmm. Could you measure it instead? This is tightly coupled with the CSS.

**Review by samuelli — APPROVED** [review]

File: iron-doc-nav.html (hunk 1 above)
Comment (aomarks) [reply iron-doc-nav.html]: Done. PTAL!

File: iron-doc-nav.html [hunk 2]
```diff
@@ -148,6 +149,23 @@
...
+          var trays = Polymer.dom(this.root).querySelectorAll('.tray');
+          for (var i = 0; i < trays.length; i++) {
+            var tray = trays[i];
+            var expanded = tray.hasAttribute('expanded');
+            if (!expanded) {
+              tray.setAttribute('expanded', '');
+            }
+            tray.style.height = 'auto';
+            var height = tray.offsetHeight;
+            tray.style.height = height + 'px';
```
Comment (samuelli) [inline iron-doc-nav.html]: inline height?

Comment (aomarks) [reply iron-doc-nav.html]: Done.

**Review by samuelli — APPROVED** [review]

**Pull request merged**